cypress/tests/data/60-content/AmwandengaSubmission.spec.js

```diff
@@ -7,47 +7,256 @@
  *
  */
 
-describe('Data suite tests', function() {
+describe('Data suite: Amwandenga', function() {
 
 	let submission;
 
 	before(function() {
+		const title = 'Signalling Theory Dividends: A Review Of The Literature And Empirical Evidence';
 		submission = {
 			id: 0,
 			section: 'Articles',
 			prefix: '',
-			title: 'Signalling Theory Dividends: A Review Of The Literature And Empirical Evidence',
+			title: title,
 			subtitle: '',
 			abstract: 'The signaling theory suggests that dividends signal future prospects of a firm. However, recent empirical evidence from the US and the Uk does not offer a conclusive evidence on this issue. There are conflicting policy implications among financial economists so much that there is no practical dividend policy guidance to management, existing and potential investors in shareholding. Since corporate investment, financing and distribution decisions are a continuous function of management, the dividend decisions seem to rely on intuitive evaluation.',
-			authors: ['Alan Mwandenga']
+			shortAuthorString: 'Mwandenga, et al.',
+			authorNames: ['Alan Mwandenga', 'Amina Mansour'],
+			assignedAuthorNames: ['Alan Mwandenga'],
+			authors: [
+				{
+					givenName: 'Amina',
+					familyName: 'Mansour',
+					email: 'amansour@mailinator.com',
+					country: 'Barbados',
+					affiliation: 'Public Knowledge Project'
+				}
+			],
+			files: [
+				{
+					'file': 'dummy.pdf',
+					'fileName': title + '.pdf',
+					'mimeType': 'application/pdf',
+					'genre': Cypress.env('defaultGenre')
+				},
+				{
+					'file': 'dummy.xlsx',
+					'fileName': 'signalling-theory-dataset.pdf',
+					'mimeType': 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
+					'genre': 'Data Set'
+				},
+				{
+					'file': 'dummy.docx',
+					'fileName': 'author-disclosure-form.docx',
+					'mimeType': 'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
+					'genre': 'Other'
+				}
+			]
 		};
 	});
 
 	it('Registers as author and creates a submission', function() {
```

Comment:
One submission test to exercise the UI, the rest to go via the API. :+1: :+1: :+1: :+1: